.prettierignore

```diff
@@ -0,0 +1,5 @@
+coverage
+.storybook
+gui/dist
+internal
```

Comment:
Possibly still prettify `internal`? There's JSON and bash in there, not just Golang.